client/src/components/FileUploadComponent/FileUpload.css

```diff
@@ -0,0 +1,32 @@
+.FileUpload {
+  height: .01rem;
+  width: .01rem;
+}
+
+.FileUpload__label {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border: solid rgba(0, 0, 0, 0);
+  border-radius: 5px;
+  font-family: var(--default-font);
+  padding: 0.30em 2.35em;
+  font-size: 15px;
+  color: var(--white);
+  background-color: var(--pink);
+}
+
+.NewUserUpload {
```

Comment:
Why is this class called `NewUserUpload` and not something related to `FileUpload`?